hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -3054,6 +3054,13 @@ func deleteControlPlaneOperatorRBAC(ctx context.Context, c client.Client, rbacNa
 	return nil
 }
 
+func deleteClusterAPIClusterRoleBinding(ctx context.Context, c client.Client, controlPlaneNamespace string) error {
+	if _, err := deleteIfNeeded(ctx, c, &rbacv1.ClusterRoleBinding{ObjectMeta: metav1.ObjectMeta{Name: "cluster-api-" + controlPlaneNamespace}}); err != nil {
```

Comment:
Can we please rename this prefix to `hypershift-cluster-api-` to avoid potential clashed in management cluster?
Can we define it as a constant in `/manifests/clusterapi/manifests` then use it here by calling deleteIfNeeded directly, no need to wrap in another no-op function.
Also I need this needs rebase since DeleteIfNeeded belongs to "github.com/openshift/hypershift/hypershift-operator/controllers/util" now

Reply:
Yes!

Reply:
Hey Alberto, your comments have been addressed and resolved. Please take a look when you have a chance.